R/zzz.R

```diff
@@ -2,5 +2,30 @@
 .onLoad <- function(...) {
   options(warnPartialMatchDollar = TRUE)
   Sys.setenv("_R_CHECK_LENGTH_1_CONDITION_" = "true")
+
+  .setEsqlabsColors()
 }
 # nocov end
+
+.setEsqlabsColors <- function() {
```

Comment:
Generating a color palette based on esqlabs colors and storing it in the `esqlabsEnv`.